lti_consumer/lti_1p3/ags.py

```diff
@@ -19,6 +19,7 @@ class LtiAgs:
     def __init__(
         self,
         lineitems_url,
+        lineitem_url,
```

Comment:
This should be an optional parameter.
`lineitems_url` is not always required.

Reply:
Updated